build.sbt

```diff
@@ -15,7 +15,7 @@ libraryDependencies ++= {
   val scalaTestV = "3.0.5"
   val bouncyCastleV = "1.57"
   val tufV = "0.6.0-1-g2f23c85"
-  val libatsV = "0.2.1-13-g4a34692"
+  val libatsV = "0.2.1-4-gd9e0738"
```

Comment:
Reverting to an older version of libats?

Reply:
It's just a snapshot that I published locally. Will be updated when we publish a version of libats from advancedtelematic/libats#95